[feat][doc] Sync 2.9.4-specific updates to 2.9.x docs (#18830)
apache/pulsar: site2/website/versioned_docs/version-2.9.x/concepts-messaging.md

# Control the ticking time for the retry of delayed message delivery,
# affecting the accuracy of the delivery time compared to the scheduled time.
# Note that this time is used to configure the HashedWheelTimer's tick time for the
# InMemoryDelayedDeliveryTrackerFactory (the default DelayedDeliverTrackerFactory).
# Default is 1 second.
delayedDeliveryTickTimeMillis=1000

# When using the InMemoryDelayedDeliveryTrackerFactory (the default DelayedDeliverTrackerFactory), whether
# the deliverAt time is strictly followed. When false (default), messages may be sent to consumers before the deliverAt
# time by as much as the tickTimeMillis. This can reduce the overhead on the broker of maintaining the delayed index

> # for a potentially very short time period. When true, messages will not be sent to consumer until the deliverAt time

Reviewer: ```suggestion
# for a potentially very short time period. When true, messages are not sent to consumers until the deliverAt time
```
